ERCS/erc-7092.md

```diff
@@ -16,7 +16,7 @@ This proposal introduces fixed-income financial bonds with key characteristics d
 
 ## Motivation
 
-Fixed-income instruments are a widely utilized asset class for corporations and other entities raising funds. However, transitioning to tokenized bonds is challenging due to existing standards like [ERC-3475](./erc-3475.md), which introduces unfamiliar concepts and leads to unnecessary gas consumption. Additionally, the lack of named variables like coupon, maturity date, and principal, makes it difficult to implement ERC-3475 since developers need to remember which metadata is assigned to each parameter.
+Fixed-income instruments are a widely utilized asset class for corporations and other entities raising funds. However, transitioning to tokenized bonds is challenging due to existing standards like [ERC-3475](./eip-3475.md), which introduces unfamiliar concepts and leads to unnecessary gas consumption. Additionally, the lack of named variables like coupon, maturity date, and principal, makes it difficult to implement ERC-3475 since developers need to remember which metadata is assigned to each parameter.
```

Comment:
Hello @lightclient,

All folders and files have been renamed _erc-Number_ in the ERCs repository, therefore referencing with the path **_./eip-3475.md_** leads to an error, that's the case now. You should revert the change to **_./erc-3475.md_**.

Let me know if I am wrong.